WolvenKit.App/Helpers/CvmDropdownHelper.cs

```diff
@@ -90,10 +91,15 @@ public static bool HasDropdownOptions(ChunkViewModel cvm)
         {
             return false;
         }
+        // Combine type check with name checks
+        if (parent.ResolvedData is gameJournalPath)
+        {
+            var validPaths = new List<string> { "path", "contact", "caller", "addressee", "briefingPath", "mappinPath" };
```

Comment:
should use a static field in the class for this or just add the strings to the switch here